### PR TITLE
Bugfix/localization - Segway Twist

### LINKED ISSUE #7: Upstream: robot_localization >2.1.7 breaks localization_local node
Placeholder for upstream issue: cra-ros-pkg/robot_localization#221

### PR DESCRIPTION
**Pull request by ClintLiddick**
Launch `robot_localization` using Segway's twist data only, not pose. Resolves #7

### CODE REVIEW

**Comment by mkoval** [comment]
N.B. This depends on https://github.com/personalrobotics/segway_rmp/pull/1.

File: config/localization_global.yaml [hunk 1]
```diff
@@ -10,23 +10,18 @@ base_link_frame: herb_base
...
 odom0_config: [
-    true,  true,  false, # x, y, z
-    false, false, true,  # roll, pitch, yaw
-    false, false, false, # x dot, y dot, z dot
-    false, false, false, # roll dot, pitch dot, yaw dot
+    false, false, false, # x, y, z
+    false, false, false,  # roll, pitch, yaw
+    true,  true, false, # x dot, y dot, z dot
+    false, false, true, # roll dot, pitch dot, yaw dot
     false, false, false  # x ddot, y ddot, z ddot
```
Comment (mkoval) [inline config/localization_global.yaml:20]: Please fix the alignment on these comments.

File: config/localization_local.yaml [hunk 2]
```diff
@@ -6,11 +6,10 @@ odom_frame: odom
...
 odom0_config: [
-    true,  true,  false, # x, y, z
-    false, false, true,  # roll, pitch, yaw
-    false, false, false, # x dot, y dot, z dot
-    false, false, false, # roll dot, pitch dot, yaw dot
+    false, false, false, # x, y, z
+    false, false, false,  # roll, pitch, yaw
+    true,  true, false, # x dot, y dot, z dot
+    false, false, true, # roll dot, pitch dot, yaw dot
     false, false, false  # x ddot, y ddot, z ddot
```
Comment (mkoval) [inline config/localization_local.yaml:14]: Please fix the alignment on these comments.

**Comment by mkoval** [comment]
:+1: LGTM besides those two formatting nitpicks.

:confetti_ball: :tada: :confetti_ball: We finally have a configuration that both makes sense and works!

**Comment by ClintLiddick** [comment]
Formatted.

**Comment by mkoval** [comment]
Looks good to me. Thanks @ClintLiddick!

Note that we'll have to build `segway_rmp` from our fork until this gets merged into upstream and the Debian packages are re-built. Here is the pull request I made again`segway_rmp`: https://github.com/segwayrmp/segway_rmp/pull/28.

**Pull request merged**